lightkurve/tests/test_periodogram.py

```diff
@@ -79,6 +78,27 @@ def test_bin():
     p = lc.to_periodogram()
     assert len(p.bin(binsize=10).frequency) == len(p.frequency)//10
 
+def test_smooth():
+    '''Test if you can smooth the periodogram and check any pitfalls'''
```

Comment:
Nitpicking: to be consistent with the PEP 8 (code style) and PEP 257 (docstring) conventions, always use double quotes in triple-quoted comments.

Reply:
Done, and replaced them elsewhere in test_periodogram() also.